tests/testthat/test-compactify.R

```diff
@@ -0,0 +1,39 @@
+library(delphi.epidata)
+library(epiprocess)
+library(data.table)
+library(dplyr)
+
+dv <- covidcast(
+  data_source = "doctor-visits",
+  signals = "smoothed_adj_cli",
+  time_type = "day",
+  geo_type = "state",
+  time_values = epirange(20211101, 20211201),
+  geo_values = "ca",
+  issues = epirange(20211129, 20211129)
```

Comment:
This test will need to be updated due to the time_value/version flip above.  This/another test should make sure that when there are multiple distinct (geo x time x other non-version-key-field) values, the LOCF-removing operations do indeed remove the right things for each (geo x time x other non-version-key-field) / that a bug doesn't cause things to fail on multiple distinct (geo x time x etc.) values.

Reply:
I updated this with a new dataset. Also, I tested it so that there are no bugs.